### ISSUE TITLE
Not getting any SMS on +49 (germany)

### ISSUE THREAD

**Issue by xShihiro** [TheSpeedX/TBomb#318]
Hey, is it normal that i dont get any SMS even if it says there are several successful ones? nothing i try to do seems to work.

**Comment by 0n1cOn3** [comment]
Yeah, certain api's responses a positive answer but they do nothing

**Comment by TheSpeedX** [comment]
Hello!

After careful consideration, I’ve decided to close all open issues for this repository. 
This project has grown into something beyond my initial expectations, but I’ve found it increasingly difficult to maintain due to low-quality contributions and spam pull requests/issues. Furthermore, I am transitioning to new ventures, and I won’t be able to continue contributing to TBomb any further.

Though I’m stepping away, I’m grateful to all those who have used and supported this tool. TBomb will remain usable through Git and PyPI, but I won’t be actively involved moving forward.

Thank you for your understanding and for using this tool!
Best regards,
TheSpeedX
